src/main.py

```diff
@@ -1,4 +1,19 @@
 import meraki
+import os
 
-print("hello world")
 
+# Defining your API key as a variable in source code is not recommended
+API_KEY =os.getenv('CAMERA_API_KEY')
```

Comment:
I would suggest using [naming from the docs](https://github.com/meraki/dashboard-api-python/#usage):
```suggestion
API_KEY = os.getenv('MERAKI_DASHBOARD_API_KEY')
```

As it seems that if the env var has the name `MERAKI_DASHBOARD_API_KEY` we do not need to use it in code anymore (it looks like it in [their readme](https://github.com/meraki/dashboard-api-python/#usage)).

Reply:
I gave you fake info, we still need to pass the API key as arguments to the methods, their examples just omit this detail.

However I would still insist on using `MERAKI_DASHBOARD_API_KEY` name.